gadget/gadget_test.go

```diff
@@ -111,8 +111,6 @@ volumes:
         type: bare
         size: 623000
         offset: 0
-        content:
```

Comment:
I think I'd leave the content entry and just create a file with some data in the test such that we encode the fact that even wit multi volume gadget the content of volumes other than system must exist in the gadget directory.

Reply:
Good point I will add this back in

Reply:
done